frontend/src/components/commons/PopupBox.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { Component } from "react";
+import { Modal } from "react-bootstrap";
+import PropTypes from "prop-types";
+
+export const BUTTON_TYPE = {
+  primary: "btn btn-primary",
+  secondary: "btn btn-secondary"
+};
+
+const styles = {
+  boxshape: {
+    borderWidth: "1px 1px 0 1px",
+    borderStyle: "solid",
+    borderColor: "#00565e",
+    borderRadius: "5px 5px 5px 5px"
+  },
+  boxContent: {
+    paddingLeft: 20,
+    paddingRight: 20
+  },
+  modalHeader: {
+    display: "table",
+    width: "100%"
+  },
+  modelTitle: {
+    position: "absolute"
+  },
+  buttonsZone: {
+    width: "100%"
+  },
+  leftBtnLocation: {
+    float: "left"
+  },
+  rightBtnLocation: {
+    float: "right"
+  },
+  buttonSize: {
+    minWidth: 125
+  }
+};
+
+class PopupBox extends Component {
+  constructor(props, context) {
+    super(props, context);
+
+    this.state = {
+      show: false
+    };
+
+    this.PropTypes = {
+      title: PropTypes.string.isRequired,
+      description: PropTypes.string.isRequired,
+      leftButtonType: PropTypes.string.isRequired,
+      leftButtonTitle: PropTypes.string.isRequired,
+      leftButtonAction: PropTypes.func,
+      rightButtonType: PropTypes.string.isRequired,
```

Comment:
Should we keep left button required? I think we will almost always have a left *or* a right button, but will we always have a left one?

Either way, I don't think the rightButtonType needs to be required.

Reply:
I think they should both be not required. You should be able to exit the dialog by clicking outside of the dialog.

Reply:
I removed the required parameters from the prop types and I added some conditions to make the button's type and the button' title required to be displayed.

See commit #https://github.com/code-for-canada/project-thundercat/pull/126/commits/7f89723d3bb5645b6afe904c9139189d0f3028c4